src/styles/components/_definition.scss

```diff
@@ -4,50 +4,99 @@ $breakpoint-background-image-size: 75rem;
   color: var(--foreground-color);
   width: 100%;
   position: relative;
-  margin: 2.5rem 0 0; // top: 40px; left&right: 0; bottom: 0;
+  padding: 8rem 0;
 
-  @media (min-width: $breakpoint-scale-up) {
-    margin-bottom: -8.5625rem; // compensates for code example padding
+  &::after {
+    content: "";
+    position: absolute;
+    top: 0;
+    left: calc(-50vw + 49.5%); // Forbidden Zastrow Knowledge
+    width: 100vw;
+    height: 100%;
     background-image: url("../../../public/images/definition_text_light.svg");
     background-size: contain;
-    background-position: center right -3rem;
     background-repeat: repeat-x;
+    mask-image: linear-gradient(to right, transparent max(50vw, 70ch), hsl(100deg 100 100));
 
     @media (prefers-color-scheme: dark) {
       background-image: url("../../../public/images/definition_text_dark.svg");
     }
   }
 
   &__container {
-    width: 100%;
     background: linear-gradient(to right, var(--background-color) 80%, transparent);
-    padding: 1.25rem;
+    padding: 0;
 
     @media (min-width: $breakpoint-background-image-size) {
       background: linear-gradient(to right, var(--background-color) 70%, transparent);
+      padding: 10.5rem 0;
```

Comment:
I think the `&__container` class got deleted from the Definition component itself. Probably safe to delete these!